docs/querying.md

```diff
@@ -51,6 +51,48 @@ Offset is using the QueryBuilder and provides the ability to offset the results.
 ```crystal
 Post.offset(100).limit(50)
 ```
+## Aggregates
+
+**NOTE:** The return type of each aggregate function can differ depending on the database adapter being used.  For example the `sum ` method:
+
+* `MySQL` will return a `Float64` even if it is an integer.
+* `Postgres` and `Sqlite` will return them a `Int64` or `Int32` depending on the size of the integer.
```

Comment:
Is there a lot of difference between adapters here? Would this documentation be worth specifying explicitly with the example of the #sum method? If so, Perhaps a table showing the return types for each method?

Reply:
Probably would be a good idea. :+1:

Reply:
Might be easier to just point the user at the specs for them to look at.  The table would be quite complex.  I don't know how to easily display the data without making a _HUGE_ table or a bunch of smaller ones, for something that can be figured out pretty quickly looking at the spec.